client/public/src/components/app.jsx

```diff
@@ -68,7 +68,7 @@ const ProductOptions = () => {
       // in one iteration, adding the colors, and quantities for each size/color
       if ( !colorsTracker[item.color] ) {
         colorsTracker[item.color] = true;
-        colors.push([item.color, item.colorUrl]);
+        colors.push([item.color, item.colorurl]);
```

Comment:
might consider using '_' to separate between words ex) color_url for better practice

Reply:
oh that's a good point. Would have to update my database column names. Will do if I have time at the end :)